ansible/install_newrelic_infra_agent.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Install New Relic infrastructure and logs
+  hosts: all
+  roles:
+    - role: newrelic.newrelic_install
+  vars:
+    targets:
+      - infrastructure
+      - logs 
+  environment:
+    NEW_RELIC_API_KEY: "{{ lookup('env', 'NEW_RELIC_API_KEY') }}"
+    NEW_RELIC_ACCOUNT_ID: "3942575"
```

Comment:
Are the services automatically updated if we change this account ID in the future?

Reply:
Can you explain to me how that works?